### ISSUE TITLE
VS Code extension ?

### ISSUE THREAD

**Issue by TanukiSharp** [melink14/rikaikun#148]
Hi @melink14 

Is there any chance you make a [VS Code](https://code.visualstudio.com/ "VS Code") extension of your wonderful Rikaikun ?
Since [VS Code](https://code.visualstudio.com/ "VS Code") is an [Electron](https://electronjs.org/ "Electron") app and fully web-tech-based, should be technically doable.

Thanks for considering.

**Comment by melink14** [comment]
Are you editing a lot of code that's in Japanese? Something like this sounds cool but I'm not sure how useful it is.

**Comment by TanukiSharp** [comment]
I also use VS Code as a text reader / editor, not necessarily code ;)
Since I leave in Japan and work with Japanese people, yes I do read/write a lot of Japanese text, so this would be super useful for me, and probably to many others.

**Comment by tora-pan** [comment]
I'm curious about this one. 

I wonder if this would be widely used?

This wouldn't  be a simple task but more of a full separate project creating a VSCode extension... 🤔

**Comment by melink14** [comment]
Yes, I'm not sure how much we could share; I assume we'd need shims around detecting hovers and showing pop ups but the rest would be the same.

It'd be cool to have something that worked in all electron apps but this one is definitely out of scope in the near term. Unless someone was motiviated and wanted guideance.
